src/components/widgets/StoicWidget/index.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from "react";
+import { useGetStoicQuote } from "./useGetStoicQuote";
+import { ReactComponent as GreekLogo } from "../../../assets/icons/greek.svg";
+import { ReactComponent as CloudLogo } from "../../../assets/icons/cloud.svg";
+import { ReactComponent as CloudAltLogo } from "../../../assets/icons/cloud-alt.svg";
+
+interface StoicData {
+  author: string;
+  quote: string;
+}
+
+export const StoicWidget = () => {
+  const [stoicData, setStoicData] = useState<StoicData[] | undefined>([]);
+  useGetStoicQuote(setStoicData);
```

Comment:
Since the hook returns multiple quotes, I suggest indicating this in the hook name by pluralizing - i.e "useGetStoicQuotes".